Adds Prettier apex

codefriar opened the pull request:
Also formats the two apex classes.

muenzpraeger commented on redwoods-insurance-salesforce/package.json:
```diff
@@ -26,7 +26,8 @@
         "husky": "^3.0.5",
         "lint-staged": "^9.3.0",
-        "prettier": "^1.18.2",
+        "prettier": "1.18.2",
```
We should allow version upgrades by prepending `^` to the version number.

muenzpraeger replied:
Also prettier related scripts in `package.json` don't have file extensions added for Apex classes and triggers.

codefriar replied:
Fair, adjusting ;)

codefriar commented:
@muenzpraeger  - I've added the ^ and added 'cls,trigger' to the file extensions list in 3 places. Manual testing with npm run prettier include the apex classes. No Triggers are (currently, or planned) included, so trigger formatting has not been tested.
